cmake/IntelMixed.cmake

```diff
@@ -0,0 +1,61 @@
+set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -g -traceback -fpp -fno-alias -auto -safe-cray-ptr -ftz -assume byterecl -sox -align array64byte -qno-opt-dynamic-align")
```

Comment:
This file is a combination of `CMAKE_C_FLAGS` from `IntelLLVM.cmake` and `CMAKE_Fortran_FLAGS` from `Intel.cmake`